src/Pages/Login/index.scss

```
@@ -0,0 +1,253 @@
+.login-page{
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  padding: 50px 0;
+
+  .login_container {
+    display: flex;
+    padding: 50px 0;
+  
+    .login-wrap {
+      display: flex;
+      min-height: 545px;
+
+      @media (max-width: 1250px){
+        padding: 59px 0;
+      }
+
+      .login_banner {
+        display: flex;
+        padding: 16px 9px 20px;
+  
+        .banner-wrap{
+          margin: 37px 40px 0 0;
+
+          @media (max-width: 1250px){
+            margin-right: 0;
+          }
+
+          @media (max-width: 1023px){
+            display: none;
+          }
+   
+          .info {
+            padding: 16px 9px 20px;
+  
+            strong {
+              font-weight: 600;
+              font-size: 25px;
+              line-height: 32px;
+              color: #252525;
+            }
+
+            .description {
+              margin-top: 5px;
+              font-size: 13px;
+              font-weight: 600;
+              line-height: 18px;
+            }
+          }
+          
+          img{
+            width: 540px;
+            object-fit: cover;
+          }
+        }
+        .form_wrap {
+          display: inline-block;
+          width: 100%;
+          max-width: 480px;
+          padding: 60px 70px;
+          font-size: 16px;
+          text-align: left;
+          border: 1px solid #e5e5e5;
+  
+          @media (max-width: 1250px) {
+            max-width: 420px;
+          }
+  
+          #logo {
+            display: inline-block;
+            line-height: 1;
+            background: url('../../images/kakao-sprite.png');
+            width: 88px;
+            height: 27px;
+            margin: 0 0 30px 0;
```

```suggestion
            margin-bottom: 30px;
```
이 경우보다는 단축 속성보다 bottom만 사용하는게 좋지 않을까요? 이와 유사한 내용들 같이 체크해주세요!

---

수정했습니다.